modules/core/shared/src/main/scala/gem/enum/GnirsWaveLengthSuggestion.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS Wavelength suggestion.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsWaveLengthSuggestion(
+  val tag: String,
+  val shortName: String,
+  val longName: String
+) extends Product with Serializable
+
+object GnirsWaveLengthSuggestion {
+
+  /** @group Constructors */ case object One extends GnirsWaveLengthSuggestion("One", "4.85", "4.85")
+  /** @group Constructors */ case object Two extends GnirsWaveLengthSuggestion("Two", "3.40", "3.40")
```

Comment:
If we ever intend to use these values as numbers we will also want numeric fields for them.